api/envoy/extensions/filters/http/oauth2/v3/oauth.proto

```diff
@@ -101,6 +101,9 @@ message OAuth2Config {
   // Optional resource parameter for authorization request
   // RFC: https://tools.ietf.org/html/rfc8707
   repeated string resources = 10;
+
+  // Disable chunked transfer encoding when requesting an access token.
+  bool disable_chunked_transfer = 11;
```

Comment:
It seems that we can always disable chunked transfer since the request isn't going to be big anyway. It makes more sense without the configuration.

Reply:
Is chunked transfer useful in the edge case? Not sure.

Reply:
I'm not an expert, but that was my reasoning too, that the request should be relatively small. At least login.microsoftonline.com apparently assumes so since it doesn't support chunked transfer.

Reply:
Even when the request is big, the code below doesn't transfer them in chunks. So it doesn't make sense to use chunk transfer anyway. We should just make the change.

Reply:
@lizan OK, should I revert the commits with the configuration changes?

Reply:
Yes, let's disable chunked transfer for all cases.

Reply:
I reverted the configuration parameter.